### ISSUE TITLE
What's the trick to compiling this on ArchLinux?

### ISSUE THREAD

**Issue by kseistrup** [avahe-kellenberger/nimdow#50]
**Describe the bug**

Every time I try to compile this on ArchLinux I end up with something like

```sh
$ nimble release --verbose
  Executing task release in …/nimdow/nimdow.nimble
Hint: used config file '/etc/nim/nim.cfg' [Conf]
Hint: used config file '/etc/nim/config.nims' [Conf]
..............................
…/nimdow/src/nimdowpkg/event/xeventmanager.nim(7, 29) Error: undeclared identifier: 'XEvent'
stack trace: (most recent call last)
/tmp/user/1234/nimblecache/nimscriptapi.nim(168, 16)
…/nimdow/nimdow_1362467.nims(28, 8) releaseTask
/usr/lib/nim/system/nimscript.nim(260, 7) exec
/usr/lib/nim/system/nimscript.nim(260, 7) Error: unhandled exception: FAILED: nim c -o:bin/nimdow -d:release --opt:speed src/nimdow.nim [OSError]
     Error: Exception raised during nimble script execution
```

I have tried with community/nim (currently 1.2.0-1) and community/nimble (currently 1:0.11.4-1), as well as with nim-git (1.3.5) and nimble-git (v0.11.4).

Am I doing something wrong?

Running `nimble build --verbose` also ends with `Error: undeclared identifier: 'XEvent'`.

**To Reproduce**

1. Clone the nimdow repo
2. cd to the nimdow directory
3. Type `nimble release --verbose` and wait

**Expected behavior**

A successful build that gives me a nimdow binary.

**Additional context**

```sh
$ uname -a
Linux home 5.7.2-arch1-1 #1 SMP PREEMPT Wed, 10 Jun 2020 20:36:24 +0000 x86_64 GNU/Linux
```
(I am aware that prebuilt binaries can be installed via AUR, but that's now what I want.)

**Comment by dakyskye** [comment]
Are you sure you have required dependencies installed? Try `nimble install` and then `nimble build`.

**Comment by kseistrup** [comment]
> Are you sure you have required dependencies installed?

Not at all, I was just following the instructions in the README: clone repo, then execute `nimble release`.

> Try `nimble install` and then `nimble build`

Same result: `Error: undeclared identifier: 'XEvent'`

_However_, when I added `--verbose` to the commands above, I noticed that nimble is looking for something in `~/.nimble/`. When I removed all the contents of that directory and started with `nimble install`, building nimdow succeeded.

However _2_: When I build nimdow from the latest `git pull`, I seemingly get nimdow@0.5.1. The nimdow binary doesn't answer to `-h`, `--help`, `-help`, `help`, `-v` or `--version`, so I can't tell what the real version is. But the releases tab here on Microsoft Github says the latest is 0.5.2. Have I run into a BLM issue so that I should choose another branch than master?
